examples/op_examples/agg.py

```diff
@@ -1,21 +1,254 @@
+import time
```

Comment:
Can we make a separate `examples/op_examples/agg_with_grouping.py` or similar, and keep the simple `agg.py`? I like having the simpler files to run basic sanity checks quickly.

Reply:
Resolved